.idea/$PRODUCT_WORKSPACE_FILE$

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove `.idea`. We don't need ide specific files in the repo. Sorry, I forgot to add this to the `.gitignore`.